### PR TITLE
Fix up headers to make ZAP scans happy

### PR DESCRIPTION
**Pull request by djmitche**
https://github.com/mozilla-services/foxsec-results/blob/master/baseline-scan/index.taskcluster.net/2017-09-25

It fixes the fail (no CSP) and one of the warnings (x-powered-by).

Thanks @jhford for fixing HSTS!

### CODE REVIEW

**Comment by jhford** [comment]
Looks good to me!

**Review by jhford — APPROVED** [review]

**Comment by djmitche** [comment]
From an email:

>   * [ ] web API responses should return `default-src 'none'; frame-ancestors 'none'; base-uri 'none'; report-uri /__cspreport__` to disallow all content rendering, framing, and report violations
> https://wiki.mozilla.org/Security/FirefoxOperations#Security_Checklist

So I'll update with those options.

**Comment by djmitche** [comment]
(released as v1.1.0)

File: README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ The configuration (here `cfg.server`) has the following options:
  * `env`: either 'development' or 'production'
  * `forceSSL`: true to redirect to https using sslify; set to true for production
  * `trustProxy`: trust headers from the proxy; set to true for production
+ * `contentSecurityPolicy`: include a CSP header with default-src: none; *default is true*
```
Comment (djmitche) [inline README.md:30]: I said default is true, but I didn't implement it..

**Pull request merged**